covid_graphs/covid_graphs/country_graph.py

```diff
@@ -2,14 +2,11 @@
 from plotly.graph_objs import Figure, Layout, Scatter
 import click
 import click_pathlib
-import dash
-import dash_core_components as dcc
-import dash_html_components as html
-from flask import Flask
 import numpy as np
 from pathlib import Path
+from typing import List
 
-from .predictions import prediction_db
+from .predictions import prediction_db, CountryPrediction
 from .country_report import CountryReport
 from .formula import Curve, XAxisType
 
```

Comment:
The constant 2 lines below (GitHub doesn't allow comments there) `PREDICTION_DATE` shouldn't be a constant but rather depend on the prediction event.

Reply:
Agreed. I did not change it yet since it needs changing the graph layout logic.